modules/docs/arrow-docs/docs/docs/integrations/kotlinxcoroutines/README.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
I don't think this is a new document, it should replace the existing one.

Reply:
Either that or github is trolling me.

Reply:
It's actually the existing one but was blank

Reply:
https://raw.githubusercontent.com/arrow-kt/arrow/0.8.2/modules/docs/arrow-docs/docs/docs/integrations/kotlinxcoroutines/README.md

that's the raw file, match the header:

```
---
layout: docs
title: kotlinx.coroutines
permalink: /docs/integrations/kotlinxcoroutines/
---
```

and re-add the option in the menu, if possible:

https://github.com/arrow-kt/arrow/blob/0.8.2/modules/docs/arrow-docs/docs/_data/menu.yml#L416-L417

Reply:
Ready, thx!